src/ComposerJson.php

```diff
@@ -128,7 +128,11 @@ private function extractAutoloadPaths(string $basePath, array $autoload, bool $i
             }
 
             foreach ($paths as $path) {
-                $absolutePath = $basePath . '/' . $path;
+                if (strpos($path, '/') === 0) {
```

Comment:
We should support even windows (the same way composer does):
- https://github.com/composer/composer/blob/main/src/Composer/Autoload/AutoloadGenerator.php#L737C27-L737C41
- https://github.com/composer/composer/blob/d3d378184bf1ae88abb1c3c9f8d0c26de5ea2744/src/Composer/Util/Filesystem.php#L544

Reply:
I found you had a `Path::isAbsolute` method so I updated the code using it.
This will avoid having multiple definition of "what is absolute path" in your code.

Should I update the implementation of isAbsolute to match the one from composer ?

Reply:
Looking at the implemention, both look very similar. Our is copied from [battle-tested nette package](https://github.com/nette/utils/blob/v4.0.4/src/Utils/FileSystem.php#L272), so I believe it should be fine to keep it. Thank you